quickstart/301-machine-learning-hub-spoke-secure/main.tf

```diff
@@ -0,0 +1,28 @@
+terraform {
+  required_version = ">=0.15.0"
```

Comment:
Why are you using 0.15 instead of 1.0?

Reply:
@grayzu, updated the versions to 1.0 and ran fmt